src/commands/cli/latestrc/build.ts

```diff
@@ -67,27 +69,27 @@ export default class build extends SfdxCommand {
     }
 
     // get the current version and implement the patch version for a default rc build
-    const repo = await PackageRepo.create({ ux: this.ux });
+    const repo = await PackageRepo.create({ ux: new Ux({ jsonEnabled: this.jsonEnabled() }) });
 
-    const nextRCVersion = repo.package.getNextRCVersion(this.flags.rctag as string, this.flags.patch as boolean);
+    const nextRCVersion = repo.package.getNextRCVersion(flags.rctag, flags.patch);
     repo.nextVersion = nextRCVersion;
 
-    this.ux.log(`starting on main and will checkout ${repo.nextVersion}`);
+    this.log(`starting on main and will checkout ${repo.nextVersion}`);
 
     // start the latest-rc build process on a clean main branch
     this.exec('git checkout main');
     this.exec('git pull');
     this.exec(`git checkout -b ${nextRCVersion}`);
 
     // bump the version in the pjson to the new latest-rc
-    this.ux.log(`setting the version to ${nextRCVersion}`);
+    this.log(`setting the version to ${nextRCVersion}`);
     repo.package.setNextVersion(nextRCVersion);
     repo.package.packageJson.version = nextRCVersion;
 
-    const only = this.flags.only as string[];
+    const only = flags.only.split(',').map((s) => s.trim());
```

Comment:
This needs to be updated after the changes to sf-plugins-core https://github.com/salesforcecli/sf-plugins-core/pull/155/files#diff-34fb7f2a65843f66e33eb600720236a297d49883585830100e0dac1150768ae5R82